backend/src/plugins/Mutes/functions/unmuteUser.ts

```diff
@@ -36,6 +36,13 @@ export async function unmuteUser(
       if (muteRole && member.roles.includes(muteRole)) {
         await member.removeRole(muteRole);
       }
+      if (existingMute?.roles_to_restore) {
+        const memberOptions: MemberOptions = {};
+        memberOptions.roles = Array.from(
+          new Set([...existingMute.roles_to_restore, ...member.roles.filter(x => x !== muteRole)]),
+        );
+        member.edit(memberOptions);
```

Comment:
Same validation as above to make sure the roles we're restoring actually exist here

Reply:
Fixed with 499853b